README.md

```diff
@@ -56,36 +98,33 @@ gaiacli version
 You should see:
 
 ```
-0.15.0-rc0-0f2aa6b
+0.16.0-dev-7ef5e90
 ```
 
 ## Genesis Setup
 
-Initiliaze `gaiad` :
+Genesis files are the starting point for the full-node to sync up with the network. In order to sync up with the correct version of the Testnet, be sure to choose the genesis file corresponding to the version of the Testnet you want to connect to.
 
-```
-gaiad init
-```
+Now that we have completed the basic SDK setup, we can start working on the genesis configuration for the chain we want to connect to.
 
-You can find the corresponding genesis files [here](https://github.com/tendermint/testnets). Then replace the `genesis.json`and `config.toml` files:
+Get the genesis transactions
 
 ```
-rm $HOME/.gaiad/config/genesis.json $HOME/.gaiad/config/config.toml $HOME/.gaiad/config/addrbook.json
+git clone github.com/cosmos/testnets
 
-wget -O $HOME/.gaiad/config/genesis.json https://raw.githubusercontent.com/cosmos/testnet/master/gaia-4000/genesis.json
+cp -a testnets/gaia-5000/. $HOME/.gaiad/config/gentx
```

Comment:
This should use absolute directories aka: $GOPATH/src/github.com/cosmos/testnets/gaia-5000/.

Reply:
Do you want to use `git clone` or `go get` as a way of cloning the repo?

I think `git clone` will be better for devs who don't GOLANG 💯

Reply:
yeah git clone makes more sense - I just use go get cause I'm laZ

Reply:
should still add the instruction and ideally use absolute paths to the clone directory